packages/ui-core/src/components/color-palette/components/color/color.component.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { fireEvent, render as rtlRender } from '@testing-library/react';
+import { Color } from './index';
+
+const testColor = '#00ff00';
+const render = (overProps: any = {}) => {
+  const props = {
+    color: testColor,
+    isDragged: false,
+    onDelete: jest.fn(),
+    toggleColorPicker: jest.fn(),
+    activeColorPicker: null,
+    onColorChange: jest.fn(),
+    colorSuggestions: [],
+    ...overProps,
+  };
+
+  const wrapper = rtlRender(<Color {...props} />);
+
+  return {
+    wrapper,
+    props,
+  };
+};
+
+describe('@keen.io/ui-core/colorPalette - <Color/>', () => {
```

Comment:
The `describe` blocks are not required as Jest actually prints automatically package name in the CLI. Some of the `tests` have `describe` blocks but we are constantly getting rid of them :)